server/src-lib/Hasura/RQL/DDL/Metadata.hs

```diff
@@ -72,86 +72,12 @@ runClearMetadata _ = do
   buildSchemaCacheStrict
   return successMsg
 
-applyQP1
```

Comment:
The logic this function does has been moved to `parseJSON` of the `Metadata` type. See `parseListAsMap` [function](https://github.com/hasura/graphql-engine/pull/6103/files#diff-7b8382ab20e441fa214586be491eb6f7ca904d8b20ed97894c16a339be45219aR154).